doc/deploy.rst

```diff
@@ -0,0 +1,81 @@
+Deploy
+======
+
+New production releases should be created every time the master branch is
+updated. The master is only updated from the develop branch.
+
+Versioning scheme
+-----------------
+
+On every merge to master, the version number in ``yaml2sbml/version.py`` should
+be incremented as described below.
+
+For version numbers, we use ``A.B.C``, where
+
+* ``C`` is increased for bug fixes,
+* ``B`` is increased for new features and minor API breaking changes,
+* ``A`` is increased for major API breaking changes.
+
+Thus, we roughly follow the versioning scheme suggested
+by the `Python packaging guide <https://packaging.python.org>`_.
+
+Create a new release
+--------------------
+
+After new commits have been added via pull requests to the develop branch,
+changes can be merged to master and a new version of yaml2sbml can be released.
+Every merge to master should coincide with an incremented version number
+and a git tag on the respective merge commit.
+
+Merge into master
+~~~~~~~~~~~~~~~~~
+
+1. create a pull request from develop to master,
+2. check that all tests pass,
+3. check that the documentation is up-to-date,
+4. adapt the version number in ``yaml2sbml/version.py`` (see above),
+5. update the release notes in ``doc/releasenotes.rst``,
+6. request a code review,
+7. merge into the origin master branch.
+
+To be able to actually perform the merge, sufficient rights may be required.
+Also, at least one review is required.
+
+Create a release on GitHub
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+After merging into master, create a new release on GitHub. This can be done
+either directly on the project GitHub website, or via the CLI as described
+in
+`Git Basics - Tagging <https://git-scm.com/book/en/v2/Git-Basics-Tagging>`_.
+In the release form,
+
+* specify a tag with the new version as specified in ``pyabc/version.py``,
```

Comment:
```suggestion
* specify a tag with the new version as specified in ``yaml2sbml/version.py``,
```
Now I also know, where you have copied that from :D (since also pypesto has the same text :D)

Reply:
dammit